crawl-ref/source/dat/des/altar/ecumenical.des

```diff
@@ -419,3 +419,29 @@ x-----x
 x-----x
 xxb+bxx
 ENDMAP
+
+NAME:   dilly_ecumenical_altar_deadguardian
+TAGS:   no_monster_gen transparent
+MONS:   plant
+SUBST: z = t11
+KITEM: E = never_decay human skeleton / gnoll skeleton / elf skeleton / dwarf skeleton
+: dgn.delayed_decay(_G, "e","hoblin skeleton / hobgoblin skeleton / kobold skeleton")
```

Comment:
Not sure what purpose these corpses have in the vault's storyline?

Reply:
The intended storyline of those corpses were about the guardian, whose its corpses never decays somehow, of faded altar and its victims killed by the guardian due to trespassing the holy(?) place and abandoned, and decays. I think I should think of how to convey the story or just discard this idea.